bdpy/dl/torch/models.py

```diff
@@ -367,3 +367,447 @@ def forward(self, z: torch.Tensor) -> torch.Tensor:
         g = self.deconv(f)
 
         return g
+
+
+class AlexNetPool5Generator(nn.Module):
+    """ From caffe pool5 generator of bvlc_reference_caffenet.
+    The model trained by DeepSim using ILSVRC dataset.
+    Provided by Alexey Dosovitskiy.
+    """
+
+    def __init__(
+            self, device: Optional[Union[str, Sequence[str]]] = None):
+        super(AlexNetPool5Generator, self).__init__()
+
+        if device is None:
+            self.__device0 = 'cpu'
+            self.__device1 = 'cpu'
+        elif isinstance(device, str):
+            self.__device0 = device
+            self.__device1 = device
+        else:
+            self.__device0 = device[0]
+            self.__device1 = device[1]
+        # input 256x6x6
+        self.Rconv6 = nn.Conv2d(256, 512, kernel_size=3, stride=1, padding=1)
```

Comment:
What is the intention behind using the capital 'R' first? If we align with Pytorch's standard norms, we should start with lowercase letters because the objects in each layer are treated like functions.